stumpy/gpu_aamp_ostinato.py

```diff
@@ -69,15 +69,17 @@ def gpu_aamp_ostinato(Ts, m, device_id=0, p=2.0):
     if not isinstance(Ts, list):  # pragma: no cover
         raise ValueError(f"`Ts` is of type `{type(Ts)}` but a `list` is expected")
 
+    # Avoid overwriting `Ts` during preprocessing
```

Comment:
I don't think this comment is necessary since `Ts_copy` is pretty self-explanatory

Reply:
Will remove it